### PR TITLE
8461 r50a improve address field

### PR DESCRIPTION
**Pull request by prit-cgi**

### CODE REVIEW

File: frontend/src/components/coverage/enrollment/ResidentDetailsWithoutPHN.vue [hunk 1]
```diff
@@ -257,6 +272,37 @@ export default {
...
+      }
+      return 'Postal / Zip Code'
+    },
+    regionOptions() {
+      if (this.mailingAddressCountry === 'United States') {
+        return this.stateOptions
+      } else if (this.mailingAddressCountry === 'Canada') {
+        return this.provinceOptions
+      }
+    },
```
Comment (AnumehaSrivastava05) [inline frontend/src/components/coverage/enrollment/ResidentDetailsWithoutPHN.vue:297]: Wireframe still shows state and province as text field. Please check with Leah

File: frontend/src/util/validators.js [hunk 2]
```diff
@@ -241,6 +262,17 @@ function validateAlpha(input, length) {
   return !/[^a-zA-Z]/.test(input)
 }
 
+/**
+ * Validate that input is allowed length and that it contains alphabets and spaces
+ */
+function validateAlphaWithSpaces(input, length) {
```
Comment (AnumehaSrivastava05) [inline frontend/src/util/validators.js:268]: What is the purpose of this method? Also, Alphanumeric validator seems not working for city and province. Please have a look.

**Review by AnumehaSrivastava05 — APPROVED** [review]
Changes looks good.

**Comment by weskubo-cgi** [comment]
Branch was merged into 8419-R50c-Improve-Address-Field.

**Pull request closed without merging**